first draft of Windows Installation Guide

emkZero opened the pull request:
WARNING: incomplete - do not use yet if you don't know what do do already

---

Das muss ich fortführen, wenn ich wieder eine Windows Test-VM habe.
@Steve2955 ich würde dich drum bitten, Fehlerkorrektur zu machen. (Natürlich kannst du auch sonst contributen (oder wie man das nennt?)) :-)

Commit:
first draft of Windows Installation Guide

WARNING: incomplete - do not use yet if you don't know what do do already

Steve2955 commented on README.md:
```diff
@@ -7,8 +7,17 @@
+Installation auf Windows:
+
+- Herunterladen der **32 Bit** Binaries von der [offiziellen Seite](https://www.libsdl.org/download-2.0.php)
+- die .tar.gz mit einer Archivsoftware öffnen (ich empfehle [7-zip](https://7-zip.org)
```
Hier hast du eine schließende Klammer nach "7-zip" vergessen

emkZero replied:
Alles gefixed, gar kein Problem 😅

Steve2955 commented on README.md:
```diff
@@ -7,8 +7,17 @@
+
+- Herunterladen der **32 Bit** Binaries von der [offiziellen Seite](https://www.libsdl.org/download-2.0.php)
+- die .tar.gz mit einer Archivsoftware öffnen (ich empfehle [7-zip](https://7-zip.org)
+- SDL2 entweder in die MinGW Ordnerstruktur integrieren ODER irgendwohin entpacken und Pfad merken
+
+Sollte SDL2 irgendwo hin entpackt worden sein, funktioniert ``FindPkgConfig`` eventuell nicht mehr
```
Ich mal versucht SDL2 in die entsprechenden Ordner von MinGW zu entpacken, konnte das Ganze aber so nicht zum laufen bekommen. Blöder Weise habe ich auch nicht unbedingt die Geduld um mich die so lange mit CMake rumzuquälen. Mein Lösungsweg über den Pfad ist zwar nicht optimal, aber solange es für mich so funktioniert, reicht mir das😅  ``FindPkgConfig`` funktioniert für mich unter Windows weiterhin nicht.

Commit:
added closing parenth. to 7z link

Commit:
(WIP) Dependencies and FindPkgConfig

How to download FindPkgConfig and its deps

Commit:
added unpacking step

just in case we need to tell the user

Commit:
Explain a little more

Steve2955 reviewed (commented):
Ich habe einfach mal schnell alles heruntergeladen und in meine MinGW Installation geworfen. Leider kein Erfolg, kann aber auch mal wieder an meiner Unfähigkeit liegen 😅 

```
CMake Error at C:/Program Files (x86)/JetBrains/CLion 2020.1.1/bin/cmake/win/share/cmake-3.16/Modules/FindPkgConfig.cmake:711 (message):
  None of the required 'SDL2_image>=2.0.0' found
Call Stack (most recent call first):
  CMakeLists.txt:11 (PKG_SEARCH_MODULE)
```

Zumindest scheint es nicht an ```FindPkgConfig``` zu liegen. So wie ich das verstehe, hat er nur SDL2 nicht gefunden, was ich aber auch in MinGW reingeworfen habe^^

ZeroPointMax commented:
Wow bist du schnell :D
Ja, das sdl2_image Ding fehlt noch, das suche ich mir noch zsm

Commit:
Added Installation Steps for SDL2 _image on MinGW

ZeroPointMax commented:
Jetzt habe ich den Link für SDL2_image hinzugefügt, den muss man auch noch installieren

Steve2955 reviewed (commented):
Scheint mir zu funktionieren. Ich schätze mal ich habe nur in irgendeinem Schritt 32- und 64 bit durcheinander gehauen...bin eigentlich davon ausgegangen, dass ich einen 32bit ```MinGW``` habe ([rust-lang/25072](https://github.com/rust-lang/rust/issues/25072)) 😅 

```cmd
C:\>gcc --version
gcc.exe (x86_64-posix-seh-rev0, Built by MinGW-W64 project) 8.1.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
Fehlermedlung beim Kompilieren in CLion:
```
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find crt2.o: No such file or directory
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmingw32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib\libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../..\libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2.dll.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib\libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2.dll.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../..\libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2.dll.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2.dll.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2_image.dll.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib\libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2_image.dll.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../..\libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2_image.dll.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2_image.dll.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmingw32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib\libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../..\libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2main.a when searching for -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lSDL2main
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2.dll.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib\libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2.dll.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../..\libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2.dll.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2.dll.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2.a when searching for -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lSDL2
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2_image.dll.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib\libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2_image.dll.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../..\libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2_image.dll.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../lib/libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2_image.dll.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: skipping incompatible C:/ProgramData/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../libSDL2_image.a when searching for -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lSDL2_image
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lkernel32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -luser32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lgdi32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lwinspool
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lshell32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lole32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -loleaut32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -luuid
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lcomdlg32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -ladvapi32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmingw32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lgcc_s
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmoldname
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmingwex
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmsvcrt
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lpthread
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -ladvapi32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lshell32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -luser32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lkernel32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -liconv
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmingw32
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lgcc_s
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmoldname
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmingwex
C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin/ld.exe: cannot find -lmsvcrt
collect2.exe: error: ld returned 1 exit status
mingw32-make.exe[3]: *** [CMakeFiles\SDL_OOP.dir\build.make:150: SDL_OOP.exe] Error 1
mingw32-make.exe[2]: *** [CMakeFiles\Makefile2:75: CMakeFiles/SDL_OOP.dir/all] Error 2
mingw32-make.exe[1]: *** [CMakeFiles\Makefile2:82: CMakeFiles/SDL_OOP.dir/rule] Error 2
mingw32-make.exe: *** [Makefile:117: SDL_OOP] Error 2
```

Edit: Ist halt jetzt nur ein bisschen blöd die ganzen Files aus MinGW wieder rauszufischen 😅

Commit:
Merge branch 'master' into untested

Commit:
added Details

ZeroPointMax commented:
@Steve2955 Ich habe eben MinGW von meiner VM gelöscht und mir ein MinGW64 via Choco installiert (um deine Umgebung zu reproduzieren). Mir scheint, dass FindPkgConfig nur für 32Bit Windows verfügbar ist (aka das originale MinGW).

Daher bleibt für diese Installationen nur der direkte Weg.
Ich werde das Dok entsprechend anpassen, inkl. dem Fallstrick, dass ``int main`` zu ``int WinMain`` werden muss

Commit:
64 Bits of Updates

* clarified differences between 32 bit and 64 bit (Windows only)
+ added proper tutorial for 64 bit MinGW (Windows only)
* refactoring to prevent documentation gore

Commit:
added architecture table

Commit:
removed redundant install step

Commit:
Explanations, Details, Love

+ added explanations
+ added a little love - mostly refactoring but also Smileys :-)
* minor changes

Commit:
Merge branch 'master' into untested

ZeroPointMax commented:
Ich habe Felix @photovoltex testen lassen und alle Fallstricke, über die er gefallen ist, mit dokumentiert - sollte nun in einem besseren Zustand sein

Steve2955 reviewed (approved):
Mir sind jetzt keine Weiteren Mängel aufgefallen, du hast alles sehr verständlich und umfangreich beschrieben.
